### PR TITLE
Clean up StartWalk

### PR DESCRIPTION
**Pull request by AJenbo**
Bin exact

### CODE REVIEW

**Comment by mewmew** [comment]
Looks great! Feel free to merge :)

File: Source/player.cpp [hunk 1]
```diff
@@ -1347,317 +1347,218 @@ void __fastcall PM_ChangeOffset(int pnum)
...
+
+	if ( !(plr[pnum]._pGFXLoad & PFILE_WALK) ) {
+		LoadPlrGFX(pnum, PFILE_WALK);
+	}
+	NewPlrAnim(pnum, plr[pnum]._pWAnim[EndDir], plr[pnum]._pWFrames, 0, plr[pnum]._pWWidth);
+
+	plr[pnum]._pdir = EndDir;
+	plr[pnum]._pVar8 = 0;
+
+	if ( EndDir == 7 ) {
```
Comment (mewmew) [inline Source/player.cpp]: Do we have a definition for the direction enum?

File: Source/player.cpp [hunk 2]
```diff
@@ -1347,318 +1347,225 @@ void __fastcall PM_ChangeOffset(int pnum)
...
+	if ( pnum == myplr ) {
+		ScrollInfo._sdx = plr[pnum].WorldX - ViewX;
+		ScrollInfo._sdy = plr[pnum].WorldY - ViewY;
+	}
+
+	dPlayer[plr[pnum].WorldX][plr[pnum].WorldY] = -1 - pnum;
+	dPlayer[px][py] = -1 - pnum;
+	plr[pnum]._pVar4 = x;
+	plr[pnum]._pVar5 = y;
+	dFlags[x][y] |= 32;
```
Comment (mewmew) [inline Source/player.cpp]: Do we have a definition for the dFlags enum? If not, replace with `0x20` as this is a bitfield.

File: Source/player.cpp (hunk 1 above)
Comment (AJenbo) [reply Source/player.cpp]: DIR_SE nice catch.

**Pull request merged**